monkey/infection_monkey/monkey.py

```diff
@@ -113,6 +116,7 @@ def __init__(self, args):
 
         self._singleton = SystemSingleton()
         self._opts = self._get_arguments(args)
+        self._server_strings = [str(s) for s in self._opts.servers]
```

Comment:
BTW, in dropper this is split only to join later. Might be a good thing to fix now, even though it's off-topic